pkg/ruler/rulestore/local/local.go

```diff
@@ -185,7 +210,7 @@ func (l *Client) loadAllRulesGroupsForUserAndNamespace(_ context.Context, userID
 	var list rulespb.RuleGroupList
 
 	for _, group := range rulegroups.Groups {
-		desc := rulespb.ToProto(userID, namespace, group)
+		desc := rulespb.ToProto(path.Base(rootDir), namespace, group)
```

Comment:
`ToProto` expects the first argument to be the userID; why are you using the path here?